synapse/models/inet.py

```diff
@@ -411,6 +410,45 @@ def _revModl201708141516(self):
         if 'syn:type:fields' not in node[1]:
             self.core.addRows([(node[0], 'syn:type:fields', 'url=inet:url,file=file:bytes', s_common.now())])
 
+    @modelrev('inet', 201708172326)
+    def _revModl201708141516(self):
+
+        props = [
+            ('user',
+             {'ptype': 'inet:netuser', 'doc': 'The netuser associated with the logon event.', 'ro': 1,
+              'form': 'inet:netlogon'}),
+            ('time',
+             {'ptype': 'time', 'doc': 'The time the netuser logged into the service', 'ro': 1,
+              'form': 'inet:netlogon'}),
+            ('status', {'ptype': 'bool', 'ro': 1,
```

Comment:
We don't need to track what the user is logging into, because this form represents something logging into the `inet:netuser` account. (thinking aloud)